pkgs/top-level/all-packages.nix

```diff
@@ -11586,6 +11586,8 @@ with pkgs;
 
   pfetch = callPackage ../tools/misc/pfetch { };
 
+  ppfetch = callPackage ../tools/misc/ppfetch { };
+  
```

Comment:
Packages residing below `pkgs/by-name` are automagically discovered and there is no need to declare them in `allpackages.nix`
```suggestion
```